src/metric/gedLib/Application.h

```diff
@@ -0,0 +1,120 @@
+#ifndef _APPLICATION_H_

```

Comment:
Is this an external library? If and library cannot be brought in 
via the git sub-module mechanism, all we can do is mention it in the README and tell users how to get it and where to put it. We just cannot store other peoples library in the repo. 

It looks like there are other source files here from gedLib. They all get the same comment as in the paragraph above.